kuma/javascript/src/search-results-page.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+import * as React from 'react';
+import { css } from '@emotion/core';
+
+import { gettext } from './l10n.js';
+import Header from './header/header.jsx';
+import Titlebar from './titlebar.jsx';
+
+import type Route from './router.jsx';
+
+type SearchResults = Array<{
+    slug: string,
+    title: string,
+    summary: string,
+    tags: Array<string>
+}>;
+
+const styles = {
+    searchResult: css({
+        padding: '8px 24px',
+        maxWidth: 600,
+        marginLeft: 'auto',
+        marginRight: 'auto'
+    }),
+    searchResultLink: css({}),
+    searchResultSummary: css({})
+};
+
+type Props = {
+    locale: string,
+    query: string,
+    data: ?SearchResults
+};
+
+export default function SearchResultsPage({ locale, query, data }: Props) {
+    return (
+        <>
+            <Header />
+            <Titlebar title={`${gettext('Search Results')}: ${query}`} />
+            {data &&
+                data.map(hit => {
+                    let url = `/${locale}/docs/${hit.slug}`;
+                    return (
+                        <div css={styles.searchResult} key={hit.slug}>
+                            <a css={styles.searchResultLink} href={url}>
+                                {hit.title}
+                            </a>
```

Comment:
I think it would be swell to display the URL of the search result here. Do you think we should add that later?

Reply:
This PR started out about the search-results-page and then turned into a bigger one about the router component. I agree that there is still work to do for the search results page. And yes, since the existing non-React version displays the URL, I'll add that as well, though maybe in a followup PR.

Also since we're removing all the checkboxes to filter searches, I'm thinking it might be useful to display document tags along with each result. ElasticSearch will return them to us, so I'll probably try adding that too. Might be useful to distinguish the HTML Canvas page from the Web API Canvas page, for example.